README.md

```diff
@@ -106,12 +102,12 @@ Each package with integration tests has their own instructions for running:
 - [Babylon.js](packages/amazon-sumerian-hosts-babylon/test/integration_test/README.md)
 - [Three.js](packages/amazon-sumerian-hosts-three/test/integration_test/README.md)
 
-### Examples Folder
-Each of the implementation packages have their own examples attached to them. Each of these files contain enough code to stand up and verify the code is working properly. To verify, open up a local server for the package you are modifying:
+### Examples and Demos
+Each of the integration packages have their own examples or demos attached to them. Each of these files contain enough code to stand up and verify the code is working properly. To verify, open up a local server for the package you are modifying:
 ```
 npm run start-{engine-name}
 ```
-Substitute `engine-name` with 'core', 'babylon' or 'three', depending on which you are testing.
+Substitute `engine-name` with 'babylon' or 'three', depending on which you are testing.
 

```

Comment:
Once again, let's mention the Babylon Demo package here.

Reply:
mentioned in a separate section as well as bablyonjs package getting started guide